demo/App.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint no-magic-numbers: 0 */
+import * as R from 'ramda';
+import React, {Component} from 'react';
+import { DataTable } from 'dash-table/index';
+import Environment from 'core/environment';
+import { memoizeOne } from 'core/memoizer';
+import Logger from 'core/Logger';
+import AppState, { AppMode } from './AppMode';
+import memoizerCache from 'core/cache/memoizer';
+
+import './style.less';
+
+class App extends Component<any, any> {
```

Comment:
Renaming to `.tsx` file but mostly just filling in with `any`. Can improve some more later. At least now it's TS too.